CHANGELOG.rst

```diff
@@ -54,6 +54,11 @@ Features
   ``intent_tokenization_flag``: Flag to check whether to split intents (default ``False``).
   ``intent_split_symbol``: Symbol on which intent should be split (default ``_``)
 
+- `#4811 <https://github.com/RasaHQ/rasa/issues/4811>`_: Support invoking a ``SlackBot`` by direct messaging or @ mentions.
```

Comment:
@kearnsw Please don't edit the `Changelog.rst` manually. Instead, create a file `4812.improvement.rst` in the `changelog` directory (the process is also described in here https://github.com/RasaHQ/rasa/blob/master/changelog/README.md )